modules/dkimVerifier.jsm

```diff
@@ -373,6 +373,8 @@ var Verifier = (function() {
 	 */
 	function verifyED25519Sig(key, str, hash_algo, signature, warnings, keyInfo = {}) {
 		let hashedStr = dkim_hash(str, hash_algo, "b64");
+		// Only SHA-256 is allowed, needed to pass tests in verifySignature
+		keyInfo.algName = "sha256";
```

Comment:
Even if maybe now further up only `ed25519-sha256` is allowed, it does not make sense to restrict it here.